### PR TITLE
Add cooling period

### PR DESCRIPTION
**Pull request by jefflau**

### CODE REVIEW

File: src/components/SingleEvent/EventInfo.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import DefaultAvatar from '../User/Avatar'
 import DepositValue from '../Utils/DepositValue'
 import { ReactComponent as DefaultEthIcon } from '../svg/Ethereum.svg'
 import { ReactComponent as DefaultPinIcon } from '../svg/Pin.svg'
+import EthVal from 'ethval'
```
Comment (hiddentao) [inline src/components/SingleEvent/EventInfo.js]: dont' need this

Comment (jefflau) [reply src/components/SingleEvent/EventInfo.js]: done

File: src/components/SingleEvent/EventInfo.js [hunk 2]
```diff
@@ -144,18 +146,25 @@ class EventInfo extends Component {
...
               .toFixed(2)}{' '}
             ETH
           </span>
-          RSVP{' '}
+          RSVP:{' '}
           <span>
             <DepositValue value={party.deposit} /> ETH
           </span>
+          <span>
+            <strong>Cooling Period: </strong>
```
Comment (makoto) [inline src/components/SingleEvent/EventInfo.js]: Why is only cooling period in strong/bold while others not?

Comment (makoto) [reply src/components/SingleEvent/EventInfo.js]: And would be nice if you can link to https://kickback.events/faq (target=blank) as we do in signup page

Comment (makoto) [reply src/components/SingleEvent/EventInfo.js]: Ok, I can sign off once the link is added

**Pull request merged**